swap_meet/vendor.py

```diff
@@ -0,0 +1,101 @@
+class Vendor: 
+    def __init__(self,inventory = None):
+        if inventory is None:
+            self.inventory = []
+        else:
+            self.inventory = inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self,item):
+    #Not sure if this is the best use of the try and except clause, wanted to attempt as practice. Alternatively, I could have written: 
+    #if item not in self.inventory: return False else: self.inventory.remove(item) return item. 
+        try:
+            self.inventory.remove(item)
+            return item
+        except ValueError:
+            return False
```

Comment:
I would have done the `if`/`else` in most code but this seems like a great place to practice exception handling.  :smile: